<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdForm</class>
 <widget class="QWidget" name="ThirdForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Взять книгу</string>
  </property>
  <widget class="QLineEdit" name="fio">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>841</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>881</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите ваши данные в формате &quot; Фамилия Имя Отчество&quot;:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>394</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="abs">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Проверить</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultat">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>821</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_thirdform">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
